fhir-term-graph/kubernetes/elasticsearch.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
```

Comment:
not sure the context, not ideal at the base of that directory.  I'd prefer to see src/main/kubernetes or in build/

Reply:
I was following the location of "docker", but will move both "docker" and "kubernetes" into src/main/